src/make_table_example.F90

```diff
@@ -150,6 +157,8 @@ program make_table_example
   Imax_logtemp = log10(150.0d0)
   Imin_logeta = log10(0.1d0)
   Imax_logeta = log10(100.0d0)
+  Imin_logn_N = 35.0d0 
+  Imax_logn_N = 37.0d0
```

Comment:
is the range 35 to 37 the intended region, or can it be bigger? is logn_N = 37 corresponds to a maximum x_n * \rho ~ 2e13 g/cm^3?